pkg/resources/configmaps.go

```diff
@@ -14,6 +16,7 @@ func GetConfigMapOrDefault(ctx context.Context, c client.Client, name types.Name
 	cm := &v1.ConfigMap{}
 	if err := c.Get(ctx, name, cm); err != nil {
 		if errors.IsNotFound(err) {
+			logrus.Debug(fmt.Sprintf("%s config not found in ns ( %s ) falling back to default strategy", name.Name, name.Namespace))
```

Comment:
`logrus.Debugf`?

Reply:
Ill catch that in the release PR, just to speed up `prow`

Reply:
Cool np